index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" href="./src/styles.css">
+    
+    <title>Document</title>
+</head>
+<body>
+    
+    <h1>-My Dog-</h1>
+    <h2>Whats up, I'm <a href="nacho">Freddie Nacho!</a></h2>
+    <p>You guessed it, I'm a dog. More specifically, I'm a <a href="https://en.wikipedia.org/wiki/Golden_Retriever">golden retreiver</a>. 
+        You're probably wondering, "how'd di you get such an awesome name?" Well my mom is obsessed with two things: </p>
+    
+    <ol>
```

Comment:
```<ol></ol>``` elements should not be used as wrappers for content unless they are ```<li></li>``` tags. This could be an easy refactor. I would also consider having more   of a commit history so I can gauge how incrementally you are working.

Reply:
hey Mack, I updated everything and passed travis.
I will make sure to have more commits in the future.